lib/services/events/webhooks/conversation_message.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * This function handles incoming "conversation:message" Webhook events from sunshine
+ * This event is fired when a user sends a message 
+ * 
+ * Actions taken:
+ *  - Create a local message in the database
+ * 
+ * Reference: https://docs.smooch.io/rest/#operation/eventWebhooks
+ *
+ * Created on Thu Mar 11 2021
+ *
+ * Copyright (c) 2021 Crossroads Foundation
+ */
+
+import { registerWebhookHandler }          from '..'
+import  db                                 from "../../../db";
+import { AuthorType }                      from "@prisma/client";
+import * as assert                         from '../../../utils/assertions'
+import {
+  ConversationMessageEvent,
+  WebhookEventBase,
+  WebhookEventType
+} from "../../../typings/webhook_types";
+
+/**
+ *
+ *
+ * @export
+ * @param {ConversationCreatedEvent} event
+ * @returns {Promise<void>}
+ */
+export async function onMessageCreated(event: WebhookEventBase) : Promise<void> {
+  // @TODO 
```

Comment:
NOTE: This is work that I stopped halfway to add factories. No need to review this yet, there will be an upcoming PR with all this feature completed